Marlin/planner.cpp

```diff
@@ -1303,8 +1321,12 @@ void Planner::set_position_mm(const AxisEnum axis, const float& v) {
 
 // Recalculate the steps/s^2 acceleration rates, based on the mm/s^2
 void Planner::reset_acceleration_rates() {
-  LOOP_XYZE(i)
+  uint32_t highest_acceleration_allaxes_steps_per_s2;
```

Comment:
>Already patched it.

Thanks for fixing.
https://github.com/MarlinFirmware/Marlin/commit/3fcf91580872bf0c7a415a8ff539e5f0cf01a244